apps/dashboard/legacy-handler.js

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+import path from 'path';
+
+export const legacyHandler = (__dirname, req, res, next) => {
```

Comment:
the shared legacy handler that is used in Vite dev server and in the proxy script that is used for the testing